build-image.sh

```diff
@@ -267,6 +268,11 @@ if [ "$IMAGE_TYPE" = "orangepipc2" ]; then
 	sudo ln -s $(cd root/boot; ls -d dtb-*-sunxi64 | head -n1) root/boot/dtb
 	unpack_deb "linux-image-dev-sunxi64.deb" "root"
 	sudo ln -s $(cd root/boot; ls -d vmlinuz-*-sunxi64 | head -n1) root/boot/Image
+elif [ "$IMAGE_TYPE" = "raspberrypi" ]; then
+  sudo 7z e -y deps/rpi-firmware-nonfree-master.zip -odeps/brcm "firmware-nonfree-master/brcm/*" > /dev/null
```

Comment:
I think you should not invoke 7z as root, only the other commands.

Reply:
Thanks for your note, fixed.